builder/builder.go

```diff
@@ -557,7 +560,8 @@ func (b *Builder) BuildUpdate(prefixflag string, minVersion int, format string,
 	}
 
 	// Ensure the format dir exists.
-	formatDir := filepath.Join(b.Statedir, "www", "version", "format"+b.Format)
+	outputDir := filepath.Join(b.Statedir, "www")
+	formatDir := filepath.Join(outputDir, "version", "format"+b.Format)
```

Comment:
why not just 
`formatDir := filepath.Join(b.Statedir, "www", "version", "format"+b.Format)` ? I don't see it being used under here.

Reply:
Oops, leftover change from before splitting out new-swupd and old-swupd code. Fixed now.